packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualCreator.sol

```diff
@@ -35,6 +35,8 @@ contract PerpetualCreator is ContractCreator, Testable, Lockable {
         address collateralAddress;
         bytes32 priceFeedIdentifier;
         bytes32 fundingRateIdentifier;
+        FixedPoint.Unsigned fundingRateRewardRate;
```

Comment:
Do we need to pass these in anymore to the Perpetual constructor?

Reply:
Ah, nice catch.

Reply:
Done.